pkg/daemon/update.go

```diff
@@ -19,6 +19,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	DefaultDirectoryPermissions os.FileMode = 0755
```

Comment:
it doesn't look like this is used anywhere?

Reply:
It does now! :) I added a commit to make sure we match Ignition there.